aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java

```diff
@@ -216,7 +234,9 @@ public void run(String... args) {
                 Files.deleteIfExists(outputFile);
             }
 
-            emailService.sendCompletedJobEmail(email, batchJobId, jobReportUrl, expirationPeriod);
+            if(emailService != null) {
+                emailService.sendCompletedJobEmail(email, batchJobId, jobReportUrl, expirationPeriod);
```

Comment:
This should probably have its own try statement rather than using the higher level one, so you can throw an error message similar to the other email.

Reply:
Will do.